src/go/k8s/pkg/resources/certmanager/admin_api.go

```diff
@@ -12,50 +12,34 @@ package certmanager
 import (
 	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
 	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/resources"
-	"k8s.io/apimachinery/pkg/types"
 )
 
 const (
 	adminAPI = "admin"
-	// AdminAPIClientCert cert name - client certificate for Admin API
-	AdminAPIClientCert = "admin-api-client"
-	// AdminAPINodeCert cert name - node certificate for Admin API
-	AdminAPINodeCert = "admin-api-node"
 )
 
-// AdminAPINodeCert returns the namespaced name for the Admin API certificate used by nodes
-func (r *PkiReconciler) AdminAPINodeCert() types.NamespacedName {
-	return types.NamespacedName{Name: r.pandaCluster.Name + "-" + AdminAPINodeCert, Namespace: r.pandaCluster.Namespace}
-}
-
-// AdminAPIClientCert returns the namespaced name for the Admin API certificate used by clients
-func (r *PkiReconciler) AdminAPIClientCert() types.NamespacedName {
-	return types.NamespacedName{Name: r.pandaCluster.Name + "-" + AdminAPIClientCert, Namespace: r.pandaCluster.Namespace}
-}
-
 func (r *PkiReconciler) prepareAdminAPI(
 	issuerRef *cmmeta.ObjectReference,
 ) []resources.Resource {
 	toApply := []resources.Resource{}
 
 	// Redpanda cluster certificate for Admin API - to be provided to each broker
-	cn := NewCommonName(r.pandaCluster.Name, AdminAPINodeCert)
-	certsKey := types.NamespacedName{Name: string(cn), Namespace: r.pandaCluster.Namespace}
+	certNames := r.CertificateNames()
+	adminAPIKey := certNames.AdminAPINodeCert
```

Comment:
Would you consider embedding `Certificates` in pki? For example, these two lines would become:
```
adminAPIKey := r.AdminAPINodeCert
```

I do have a slight preference towards the current approach of having functions like `AdminCert()` that add the suffix instead of having it around the code base, but not a big deal.